harness/determined/cli/_util.py

```diff
@@ -100,7 +101,14 @@ def setup_session(args: argparse.Namespace) -> api.Session:
     master_url = args.master or util.get_default_master_address()
     cert = certs.default_load(master_url)
 
-    return api.Session(master_url, args.user, authentication.cli_auth, cert)
+    retry = urllib3.util.retry.Retry(
```

Comment:
Something will need to be dropped into `determined.Determined.__init__` or similar, too.